packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -93,7 +117,7 @@ const getParticipantsForDefaultRender = (
   participants: CommunicationParticipant[],
   excludeMe: boolean,
   myUserId: string | undefined
-) => {
+): CommunicationParticipant[] | [] => {
```

Comment:
Does this need the `| []`? better to be strongly typed if possible. I see there's a 
```
if (participants === undefined) {
  return [];
}
```
But participants can't be undefined as a parameter so that should be able to be removed or maybe updated to:
```
if (participants.length === 0) {
  return [];
}
```
If that is what the check is meant to be for

Reply:
Yeah, please tighten this type.

I think you might just need to change line 122 to be `return [] as CommunicationParticipant[]` so typescript knows to type the empty list correctly.